src/Patterns/Path.php

```diff
@@ -69,24 +69,24 @@ class Path implements PathType
     private array $types = [];
 
     /**
-     * @var Variable
+     * @var ?Variable
```

Comment:
`?Variable` is not a valid PHPDoc type hint, it should be `Variable|null`.

Reply:
Resolved in 342cb91